static/css/restyle.css

```diff
@@ -0,0 +1,695 @@
+/* Hide outlines Firefox shows on every click */
+* {
+  outline: none;
```

Comment:
This will make navigation impossible for someone who uses a keyboard. Try it by tabbing through the links. At least the current site does somewhat cater to that case.